src/components/Common/StyledLink.js

```diff
@@ -26,17 +26,31 @@ export const StyledLinkCss = css`
     margin-top: ${remcalc(6)};
     background: ${({ theme }) => theme.colors.text};
     box-sizing: border-box;
+    transition: all ${props => props.theme.animations.fast} ease-out;
```

Comment:
`${({ theme }) => theme.animations.fast} ease-out;` to keep it consistent with the rest of the file